content/agent/kubernetes/metrics.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Kubernetes Metrics
+kind: documentation
+---
+
+#### Kubernetes
+{{< get-metrics-from-git "kubernetes" >}}
+
+#### Kubelet
+{{< get-metrics-from-git "kubelet" >}}
+
+#### Kubernetes State
```

Comment:
Kubernetes State should be `kube-state-metrics`, that's the official name of this piece of software.